src/main/config/messages_template.xml

```diff
@@ -7,413 +7,415 @@ Copyright 2005, 2006 IBM Corporation
-    <response></response>
+    <reason>Invalid @xml:lang attribute value '%1'.</reason>
+    <response>Check the correct value for the target language.</response>
   </message>
-  
+
   <message id="DOTJ057E" type="ERROR">
-    <reason>The id attribute value "%1" is not unique within the topic that contains it.</reason>
+    <reason>The @id attribute value '%1' is not unique within the topic that contains it.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTJ058E" type="ERROR">
-    <reason>Both %1 and %2 attributes defined.</reason>
+    <reason>Both the @%1 and @%2 attributes are defined.</reason>
     <response>A single element may not contain both generalized and specialized values for the same attribute.</response>
   </message>
-  
+
   <message id="DOTJ059E" type="ERROR">
-    <reason>Invalid key scope name "%1".</reason>
+    <reason>Invalid key scope name '%1'.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTJ060W" type="WARN">
-    <reason>Key "%1" was used in conkeyref but is not bound to a DITA topic or map.</reason>
-    <response>Cannot resolve conkeyref value "%2" as a valid conref reference.</response>
+    <reason>The '%1' key is used in a @conkeyref attribute but it is not bound to a DITA topic or map.</reason>
+    <response>Cannot resolve '%2' as a valid content key reference.</response>
   </message>
 
   <message id="DOTJ061E" type="ERROR">
-    <reason>Topic reference target is a DITA map but format attribute has not been set.</reason>
-    <response>Set format attribute value to "ditamap".</response>
+    <reason>Topic reference target is a DITA map but the @format attribute has not been set.</reason>
+    <response>Set the @format attribute value to 'ditamap'.</response>
   </message>
-    
+
   <message id="DOTJ062E" type="ERROR">
-    <reason>Invalid %1 attribute value "%2".</reason>
+    <reason>Invalid @%1 attribute value '%2'.</reason>
     <response></response>
   </message>
 
   <message id="DOTJ063E" type="ERROR">
-    <reason>The cols attribute is "%1" but number of colspec elements was %2.</reason>
-    <response></response>
+    <reason>The @cols attribute is set to '%1' but the number of &lt;colspec&gt; elements was '%2'.</reason>
+    <response>Check the number of columns in the table.</response>
   </message>
 
   <message id="DOTJ064W" type="WARN">
-    <reason>Chunk attribute uses both "to-content" and "by-topic" that conflict with each other.</reason>
-    <response>Ignoring "by-topic" token.</response>
+    <reason>The @chunk attribute uses both 'to-content' and 'by-topic' that conflict with each other.</reason>
+    <response>Ignoring the 'by-topic' token.</response>
   </message>
 
   <message id="DOTJ065I" type="INFO">
-    <reason>Branch filter generated topic %1 used more than once.</reason>
-    <response>Renaming %1 to %2.</response>
+    <reason>Branch filter generated topic '%1' used more than once.</reason>
```

Comment:
Not sure, but is this a place where we should have "is" in the message ... "topic X *is* used"